shared/src/libs/helpers/date/get-relative-date/get-relative-date.helper.ts

```diff
@@ -0,0 +1,21 @@
+import { formatRelative } from 'date-fns';
+import enGB from 'date-fns/locale/en-GB/index.js';
```

Comment:
I tried to make just `date-fns/locale` import, but after restarting backend side I got this error. The most funny and strange that while changing import during hot reload everything worked fine. If someone knows the reason, pls leave a comment)

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/757a6d11-8db4-4c3a-9baa-05cddc9006ca)

Reply:
I am not sure, but I think it could be because your folder path includes letters is named not in English

Reply:
I had such problems in the past with other things